src/rpc/mempool.cpp

```diff
@@ -669,6 +670,136 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan submitpackage()
+{
+    return RPCHelpMan{"submitpackage",
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node (-regtest only).\n"
+        "\nThe package will be validated and accepted to mempool if it passes consensus and mempool policy rules. Throws a JSONRPCError if anything goes wrong.\n"
```

Comment:
"anything goes wrong" is  quite vague. Does it mean "if the transaction or package is rejected for consensus or policy reasons"?

Reply:
Hopefully now clearer